### DISCUSSION TITLE
How to create 320x200 fullscreen app ?

### DISCUSSION THREAD

**Discussion by elsasslegend** [pixijs/pixijs#7525] [Q&A]
Hello,

In my old days I was used to create some small games using Qbasic 4.5.
There was a mode called SCREEN 13 that allows drawing some graphics (points, lines) on a 300x200 fullscreen renderer.
What is the easiest way to create a PIXI app that autoscales everything into fullscreen like if we would work on a 300x200 screen ?

Thanks

**Comment by bigtimebuddy** [comment]
For general questions, please create a discussion. Thank you. Transferring this...

**Reply by eXponenta** [reply]
@djflex68  
Set renderer size as 320x200 and scale canvas via CSS to 100% of window size and disable smoothing.

https://www.pixiplayground.com/#/edit/qmqx3XFB9yrqvfD_FzLWC
